app/(root)/interview/page.tsx

```
@@ -11,7 +11,7 @@ const Page = async () => {
       <Agent
         userName={user?.name!}
         userId={user?.id}
-        profileImage={user?.profileURL}
+        profileImage={user?.profileUrl!}
```

ESLint

[error] Optional chain expressions can return undefined by design - using a non-null assertion is unsafe and wrong.

(@typescript-eslint/no-non-null-asserted-optional-chain)